blenderproc/python/types/LightUtility.py

```diff
@@ -62,6 +63,105 @@ def set_type(self, type: str, frame: int = None):
         self.blender_obj.data.type = type
         Utility.insert_keyframe(self.blender_obj.data, "type", frame)
 
+    def setup_as_projector(self, pattern: np.ndarray, frame: int = None):
+        """ Sets a spot light source up as projector.
```

Comment:
Can we add a warning here, that this should be done after the camera parameters were set.